src/processes.c

```diff
@@ -736,6 +741,14 @@ static void ps_submit_proc_list (procstat_t *ps)
 	vl.values_len = 2;
 	plugin_dispatch_values (&vl);
 
+	if (ps->num_fd != -1)
```

Comment:
I think zero (`0`) is a better "not valid" value, because `-1` will likely cause warnings / errors with some compilers (always true due to range of variable, implicit cast at assignment, …). A process without *any* file descriptors is unable to communicate with the outside world and is therefore … rare.